package.json

```diff
@@ -28,7 +28,6 @@
     "test:coverage": "yarn build && npm test -- --no-watchman --coverage",
     "test:ci": "yarn test -- --ci --no-watchman --maxWorkers=10",
     "test:coverage:ci": "yarn test:ci -- --coverage && codecov",
-    "release": "yarn check-gh-auth-token && cross-env NODE_ENV=cli yarn build && lerna publish --registry=https://registry.npmjs.org/ && yarn build:changelog && yarn push:changelog && yarn docs:publish",
```

Comment:
We should also remove those other scripts/commands that are not necessary anymore, as well as dependencies (e.g. `lerna-changelog`).

Reply:
I have removed all commands associated with lerna changelog.

Reply:
What about the `yarn docs:publish` command? Do you still need it?